checks_linux.go

```diff
@@ -210,8 +210,9 @@ func (c cond) PermissionIs() (bool, error) {
 func (c cond) ProgramInstalled() (bool, error) {
 	c.requireArgs("Name")
 	result, err := cond{
-		Cmd: "dpkg -s " + c.Name,
-	}.Command()
+		Cmd:   "dpkg -s " + c.Name,
+		Value: " install",
```

Comment:
what does this change do?

Reply:
this question also applies to line 215

Reply:
Yea lol I think that check was broken. After removing something with apt, dpkg just shows the status as deinstall. Does not return nonzero value for `-s`